src/theme/material/text-input.m.css

```diff
@@ -106,3 +106,7 @@
 .input::-ms-clear {
 	display: none;
 }
+
+.input::placeholder {
```

Comment:
input placeholders aren't really valid for material themed widgets so I'm not sure that we need this here